src/state/management/userSettings.js

```diff
@@ -0,0 +1,30 @@
+// Initial state
+const INITIAL_STATE = {
+  isDarkModeEnabled: false,
+};
+
+// Types
+export const TOGGLE_DARK_MODE = 'TOGGLE_DARK_MODE';
+
+// Actions
+export const toggleDarkMode = (isDarkModeEnabled) => {
+  return {
+    type: TOGGLE_DARK_MODE,
+    isDarkModeEnabled: isDarkModeEnabled,
+  };
+};
```

Comment:
```suggestion
export const toggleDarkMode = () => {
  return {
    type: TOGGLE_DARK_MODE,
    isDarkModeEnabled: !isDarkModeEnabled,
  };
};
```

maybe toggle?

Reply:
I debated this for 10 seconds when making this. I think passing in a boolean is better for consistency with DB level (ie. default value comes from DB)